conf/build.manifest

```diff
@@ -164,7 +175,7 @@
 		{
 			"name": "truenas_installer",
 			"repo": "https://github.com/truenas/truenas-installer",
-			"branch": "master"
+			"branch": "custom-kernel-build"
```

Comment:
Will these be changed to master later? If you only want them to test with the pull-request, you can do so just by ensuring that the branch name for this PR is the same as the branch name on those repos. Jenkins / builder will DTRT in that case, and you won't need to modify this branch manually after you merge them all back to respective master.